pkg/util/mount/mount_linux.go

```diff
@@ -727,13 +727,13 @@ func GetSELinux(path string, mountInfoFilename string) (bool, error) {
 
 func (mounter *Mounter) GetMountRefs(pathname string) ([]string, error) {
 	pathExists, pathErr := PathExists(pathname)
-	if !pathExists {
```

Comment:
@jingxu97, @msau42 I think order of these if/else statements was not correct with the unchanged `PathExists` method. Because `PathExists` method returns `false, err` if a not corrupted error occurred, so checking `pathErr != nil` statement after `!pathExists` has no effect.